src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -91,6 +98,18 @@
 
     private final Map<Project, List<RepositoryInfo>> repository_map = new TreeMap<>();
     private final Map<Project, Set<Group>> project_group_map = new TreeMap<>();
+    
+    private static final String MESSAGES_MAIN_PAGE_TAG = "main";
+    /*
+    initial capacity - default 16
+    initial load factor - default 0.75f
```

Comment:
if they are public constants, then take them from there (just in case JDK will change defaults ... )